### ISSUE TITLE
Error: Cannot find name 'Observable'

### ISSUE THREAD

**Issue by Pazitos10** [souzadriano/ionic2-google-places-autocomplete#1]
Hi there! I'm trying to use ionic-google-places along with ionic 3.6.0 and when I run: ``` ionic serve ``` I'm having the following error:

![bug](https://user-images.githubusercontent.com/5199304/29086086-c083825a-7c47-11e7-8c67-55f1862d59e2.png)

Here's my ``` ionic info ``` command output:

    cli packages: (~/my_project/node_modules)                                                                       
                                                                                                                                                      
        @ionic/cli-plugin-cordova       : 1.6.2                                                                                                           
        @ionic/cli-plugin-ionic-angular : 1.4.1                                                                                                           
        @ionic/cli-utils                : 1.7.0                                                                                                           
        ionic (Ionic CLI)               : 3.7.0                                                                                                           
                                                                                                                                                      
    global packages:                                                                                                                                      
                                                                                 
        Cordova CLI : 7.0.1 

    local packages:

        @ionic/app-scripts : 1.3.0
        Cordova Platforms  : android 6.2.3
        Ionic Framework    : ionic-angular 3.6.0

    System:

        Android SDK Tools : 26.0.2
        Node              : v8.2.1
        OS                : Linux 4.12
        npm               : 5.3.0 


I think its caused by this import (in that file):
```ts
    import { Observable } from 'rxjs/observable';
```
A temporary solution for me was to manually modify that file importing 'Rx' instead of 'observable' so the error message dissapear, but I was hoping you could solve it in the source code.

```ts 
    import { Observable } from 'rxjs/Rx';
```

Thanks in advance!

**Comment by souzadriano** [comment]
Thank you for reporting.

I will fix it soon to be compatible with ionic 3.6.

**Comment by souzadriano** [comment]
Fixed on version 1.0.1 with:

`import { Observable } from 'rxjs/Rx';`

Thank you @Pazitos10 

Closing
